.travis.yml

```diff
@@ -7,21 +7,42 @@ jobs:
      python: 2.7
      install: python scripts/dev_setup.py
      script: scripts/build.sh
+     addons:
+      apt:
+        packages:
+         - libboost-python-dev
    -
      env: CODE_COVERAGE="True"
      python: 3.5
      install: python scripts/dev_setup.py
      script: scripts/build.sh
+     addons:
+      apt:
+        packages:
+         - libboost-python-dev
    -
      env: CODE_COVERAGE="True"
      python: 3.6
      install: python scripts/dev_setup.py
      script: scripts/build.sh
+     addons:
+      apt:
+        packages:
+         - libboost-python-dev
    -
      python: 2.7
      install: pip install -qqq virtualenv
      script: scripts/package_verify.sh
+     addons:
+      apt:
+        packages:
+         - libboost-python-dev
+
    -
      python: 3.6
      install: pip install -qqq virtualenv
-     script: scripts/package_verify.sh
\ No newline at end of file
+     script: scripts/package_verify.sh
+     addons:
+      apt:
+        packages:
+         - libboost-python-dev
```

Comment:
This would mean for iot commands to work, `libboost-python-dev` is required?

Reply:
Only the new commands introduced (including refactor of existing 'iot device message send'. The reason is that it is a dependency for the [IoT SDK](https://github.com/Azure/azure-iot-sdk-python/blob/master/doc/python-devbox-setup.md#pypi-wheels) to work. Linux currently needs at least boost-python 1.58 and Windows needs the Visual C++ 2015 redistributable.

That being said I haven't figured out why newer libboost packages are not installing, even though it is [whitelisted ](https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-trusty)for travis-ci.